src/js/components/award/SummaryBar.jsx

```diff
@@ -0,0 +1,107 @@
+/**
+ * SummaryBar.jsx
+ * Created by Emily Gullo 01/20/2017
+ **/
+
+import React from 'react';
+import moment from 'moment';
+import * as Icons from '../sharedComponents/icons/Icons';
+import InfoSnippet from './InfoSnippet';
+
+const propTypes = {
+    selectedAward: React.PropTypes.object
+};
+
+export default class SummaryBar extends React.Component {
+    constructor(props) {
+        super(props);
+
+        this.state = {
+            status: ""
+        };
+
+        this.getStatus = this.getStatus.bind(this);
+    }
+
+    componentWillMount() {
+        this.getStatus(this.props.selectedAward);
+    }
+
+    componentWillReceiveProps(nextProps) {
+        if (nextProps.selectedAward.id !== this.props.selectedAward.id) {
+            this.getStatus(nextProps.selectedAward);
+        }
+    }
+
+    getStatus(award) {
+        const awardStart = moment(award.period_of_performance_start_date, 'MM-DD-YYYY');
+        const awardEnd = moment(award.period_of_performance_current_end_date, 'MM-DD-YYYY');
+        const current = moment();
+        let progress = "";
+        if (current.isBefore(awardStart, 'day') || current.isSame(awardStart, 'day')) {
```

Comment:
[http://momentjs.com/docs/#/query/is-same-or-before/](http://momentjs.com/docs/#/query/is-same-or-before/)

[http://momentjs.com/docs/#/query/is-same-or-after/](http://momentjs.com/docs/#/query/is-same-or-after/)